venia-integration-tests/src/assertions/categoryPage/index.js

```diff
@@ -110,10 +111,15 @@ export const assertProductIsInGallery = productName => {
  */
 export const assertProductIsInProductSuggestion = (
     productName,
-    productHref
+    productHref,
+		wait = 4000
 ) => {
-    cy.wait(4000);
     cy.get(searchBarSuggestedProduct).should('contain', productName);
+    cy.get(searchBarSuggestedProduct)
+        .should('be.visible')
+        .then(() => {
+            cy.wait(wait);
```

Comment:
Why do we need to wait?
Can't we check if the search bar contains something?

Reply:
I've tried to listen to a GraphQL request, as I tried to assert search results `be.visible`, but I've not success indeed. The cy.wait was used on `./src/support/visitPageCommand.js`, so I think it's a good solution at all.